website/static/js/fangorn.js

```diff
@@ -2978,9 +2978,10 @@ tbOptions = {
                 maxSize = item.data.accept.maxSize;
                 if (size > maxSize) {
                     displaySize = Math.round(file.size / 10000) / 100;
```

Comment:
Might want to use `$osf.humanFileSize` here so you don't have some odd number of MB like 3200 MB instead of 3.2 GB.